ginga/rv/plugins/TVMask.py

```diff
@@ -318,7 +318,7 @@ def recreate_toc(self):
     def _rgbtomask(self, obj):
         """Convert RGB arrays from mask canvas object back to boolean mask."""
         dat = obj.get_image().get_data()  # RGB arrays
-        return dat.sum(axis=2).astype(np.bool)  # Convert to 2D mask
+        return dat.sum(axis=2).astype('bool')  # Convert to 2D mask
```

Comment:
I think you missed one.
```suggestion
        return dat.sum(axis=2).astype(bool)  # Convert to 2D mask
```